### PR TITLE
Fix agreement reference number - cp to agreement

### PR DESCRIPTION
**Pull request by robertavram**
Fixes unicef/etools-issues#452

### CODE REVIEW

File: EquiTrack/partners/serializers/agreements_v2.py [hunk 1]
```diff
@@ -149,6 +150,7 @@ class AgreementCreateUpdateSerializer(serializers.ModelSerializer):
     partner_name = serializers.CharField(source='partner.name', read_only=True)
 
     amendments = AgreementAmendmentCreateUpdateSerializer(many=True, read_only=True)
+    country_programme = serializers.PrimaryKeyRelatedField(queryset=CountryProgramme.objects.all(), required=True)
```
Comment (ntrncic) [inline EquiTrack/partners/serializers/agreements_v2.py]: why is this needed on amendment type?

Comment (robertavram) [inline EquiTrack/partners/serializers/agreements_v2.py]: It's not on type, it's on the agreement

Comment (robertavram) [inline EquiTrack/partners/serializers/agreements_v2.py]: Although, if the type is MOU, I believe this is not required. I'll have to double check with the spec.

Comment (ntrncic) [inline EquiTrack/partners/serializers/agreements_v2.py]: according to specs it says show country programme only for PCA.

Comment (robertavram) [inline EquiTrack/partners/serializers/agreements_v2.py]: ok.. I'll fix it and add some tests
